canvas_modules/common-canvas/src/toolbar/toolbar.jsx

```diff
@@ -196,6 +196,7 @@ class Toolbar extends React.Component {
 					<ToolbarDividerItem
 						key={"toolbar-item-key-" + i}
 						overflow={overflow}
+						buttonSize={this.props.size}
```

Comment:
You don't seem to have modified ToolbarDividerItem  so what is this prop being passed in for?

Reply:
Removed it.